app/scripts/directives/processTemplateDialog.js

```diff
@@ -137,7 +138,7 @@
     };
 
     $scope.$on('templateInstantiated', function(event, message) {
-      ctrl.selectedProject = message.project;
+      ctrl.templateProject = message.project;
```

Comment:
@jhadvig There are two different projects. We need to make sure we're not getting them mixed up:

- `templateProject` -- the source project the template is from
- `selectedProject` -- the target project the template is instantiated into

You can selected a project to instantiate into on the landing page, but current project is always used when showing the dialog inside the project.

I think this is probably a two line fix: calling `ctrl.templateProjectChange()` on init and setting `currentProject` as the project in the view.

Reply:
@spadgett we need also the change the `process-template-select` so it uses the `ctrl.selectedProject` not to use the `ctrl.templateProject`, cause we are not even initiating that var